appsettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ConnectionStrings": {
+    "PostgresSQL": "Server=localhost;Database=qs;Port=5432;User Id=postgres;Password=password;"
```

Comment:
Why is this connection string name different from the one in appsettings.Development.json. This will definitely cause null reference exceptions.